public/index.html

```diff
@@ -19,6 +19,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+
+    <!--
+      Add Icons
+    -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
You don't need to do the following, you can import directly as follows: `import { AccessAlarm, ThreeDRotation } from '@material-ui/icons';`.

See: https://material-ui.com/style/icons/

Reply:
Thanks, will update